website/mailing_list/views.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    must_not_be_registration,
+)
+from website.util.permissions import ADMIN
+from website.mailing_list import utils
+
+
+###############################################################################
+# View Functions
+###############################################################################
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def get_node_mailing_list(node, auth, **kwargs):
+    return format_node_data_recursive([node], auth.user)
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def enable_mailing_list(node, **kwargs):
+    utils.celery_create_list(node._id)
+    node.mailing_enabled = True
+    node.save()
+    return {'message': 'Successfully enabled mailing lists', 'success': True}
+
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def disable_mailing_list(node, **kwargs):
+    utils.celery_delete_list(node._id)
+    node.mailing_enabled = False
+    node.save()
+
+unsubscribe_user = utils.unsubscribe_user_hook
+log_message = utils.log_message
+
+def format_node_data_recursive(nodes, user):
+    items = []
+
+    for node in nodes:
+
+        can_read = node.has_permission(user, 'read')
```

Comment:
In place of hard-coded values, could you please use the named constant? (several places throughout project)

`from website.util.permissions import ADMIN, READ` or somesuch.